internal/common/baseclient/iifo.go

```diff
@@ -0,0 +1,157 @@
+package baseclient
+
+import (
+	"math"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+// Disaster defines the disaster location and magnitude.
+// These disasters will be stored in PastDisastersDict (maps round number to disaster that occurred)
+// TODO: Agree with environment team on disaster struct representation
+type DisasterInfo struct {
+	CoordinateX float64
+	CoordinateY float64
+	Magnitude   float64
+	TurnNumber  int
+}
+
+// PastDisastersDict is a helpful construct for
+type PastDisastersDict = map[int]DisasterInfo
+
+var ourPredictionInfo shared.PredictionInfo
```

Comment:
why is this package level? shouldn't this be contained within `BaseClient`?

Reply:
Just because I missed it when updating to the new structure, will update

Reply:
Any reason for a map, and what is the key in this case? The season? I think a sequential collection works better personally

Reply:
Yeah agreed actually

Reply:
is this actually resolved?

Reply:
Changed to a list and moved the predictionInfo to baseclient